_3_design_patterns/src/main/java/code/_3_in_class/Main.java

```diff
@@ -1,8 +1,9 @@
 package code._3_in_class;
 
-public class Main {
+public class HelloWorld {
```

Comment:
Modificarile de cod aduse de tine se gasesc in folderul "_3_design_patterns" care este destinat saptamanii 3 de curs. 
Rugamintea este sa faci modificarile necesare doar in folderul saptamanii de curs, in cazul nostru: "_1_basics"